### DISCUSSION TITLE
Formily 设计器是否可以加上Button和FormButtonGroup类型的元素？

### DISCUSSION THREAD

**Discussion by iscmcloud** [alibaba/formily#1636] [Ideas]
1.formily设计器，可以增加支持Markup Schema导出吗，如果可以的话，将显著提高开发效率
2.设计器可以把Button和FormButtonGroup这些类型加上吗，因为有些场景下，按钮布局也需要

**Comment by janryWang** [comment]
1. 这个会支持的
2. 不会支持，因为加上按钮就给用户感觉这个是个页面搭建器了，还得支持事件管理，请求管理，全局状态管理等等，完全是一个App级别的产品了，其实它只是维护json-schema的工具而已
